addons/binding/org.openhab.binding.mqtt/src/main/java/org/openhab/binding/mqtt/internal/ssl/PinningSSLContextProvider.java

```diff
@@ -42,7 +42,7 @@ public PinningSSLContextProvider(PinTrustManager pinTrustManager) {
     @Override
     public SSLContext getContext() throws ConfigurationException {
         try {
-            SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+            SSLContext sslContext = SSLContext.getInstance("TLSv1");
```

Comment:
Version 1.2 was there on purpose. Everything below is not secure anymore. The version is only a minimum, so 1.3 would be fine as well.